patches/swiftshader/revert_replace_target_cpu_with_current_cpu_in_build_gn_files.patch

```diff
@@ -0,0 +1,218 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: deepak1556 <hop2deep@gmail.com>
+Date: Mon, 2 Mar 2020 00:42:24 -0800
+Subject: Revert "Replace target_cpu with current_cpu in Build.gn files"
+
+This reverts commit 847856089dc3f740f46f6f6af1be4c7cde50cdb2.
```

Comment:
Temporary change to verify the regression from https://swiftshader-review.googlesource.com/c/SwiftShader/+/41528